packages/primitives/src/svg/index.js

```diff
@@ -82,23 +82,26 @@ export const LinearGradient = ( props ) =>
  */
 export const Stop = ( props ) => createElement( 'stop', props );
 
-/**
- *
- * @param {SVGProps} props isPressed indicates whether the SVG should appear as pressed.
- *                         Other props will be passed through to svg component.
- *
- * @return {JSX.Element} Stop component
- */
-export const SVG = ( { className, isPressed, ...props } ) => {
-	const appliedProps = {
-		...props,
-		className:
-			classnames( className, { 'is-pressed': isPressed } ) || undefined,
-		'aria-hidden': true,
-		focusable: false,
-	};
+export const SVG = forwardRef(
+	/**
+	 * @param {SVGProps}                           props isPressed indicates whether the SVG should appear as pressed.
+	 *                                                   Other props will be passed through to svg component.
+	 * @param {import('react').Ref<SVGSVGElement>} ref   The forwarded ref to the SVG element.
```

Comment:
Same re. `ForwardedRef`

```suggestion
	 * @param {SVGProps}                                    props isPressed indicates whether the SVG should appear as pressed.
	 *                                                            Other props will be passed through to svg component.
	 * @param {import('react').ForwardedRef<SVGSVGElement>} ref   The forwarded ref to the SVG element.
```